src/payment/cart/Offers.jsx

```diff
@@ -60,7 +60,11 @@ function Offer({
 Offer.propTypes = {
   benefitType: PropTypes.oneOf(['Percentage', 'Absolute']).isRequired,
   benefitValue: PropTypes.number.isRequired,
-  provider: PropTypes.string.isRequired,
+  provider: PropTypes.string,
```

Comment:
We missed this - now that provider isn't required, it was throwing an error here when we didn't provide it.

Reply:
thanks!